src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10786,48 +10813,59 @@ s32 func_808430E0(Player* this) {
     }
 
     if (!(this->stateFlags1 & PLAYER_STATE1_8000000)) {
-        sPlayerCurrentFloorType = FLOOR_TYPE_0;
+        sPlayerFloorType = FLOOR_TYPE_0;
     }
-    this->unk_B6C = 0;
-    this->unk_B6E = 0;
-    D_80862B28 = 0;
+    this->floorPitch = 0;
+    this->floorPitchAlt = 0;
+    sPlayerFloorPitchShape = 0;
     return true;
 }
 
-void func_80843178(PlayState* play, Player* this) {
-    u8 spC7 = 0;
+/**
+ * Performs various tasks related to scene collision.
+ *
+ * This includes:
```

Comment:
```suggestion
 * Performs the following tasks related to scene collision:
 *
```
Just my thoughts on stating what happens explicitly rather than generally.

Reply:
Oh, forgot to bump this as well :)